### ISSUE TITLE
Error installing via npm and bundling

### ISSUE THREAD

**Issue by evanshortiss** [feedhenry/fh-js-sdk#210]
I installed this module via npm into a project and then I tried to create my vendor bundle like so:

```
browserify -o www/dist/vendor.js -r fh-js-sdk -r react -r react-redux -r redux
```

Doing so results in an error like so being generated when browserify tries to package in `fh-js-sdk` due to the transforms required: 

```
Error: Cannot find module 'browserify-shim' from '/workspaces/fh/my-app/node_modules/fh-js-sdk'
```

Currently I've found two solutions:

1. Install `browserify-shim` in my project
2. Install `browserify-shim` inside the `node_modules` of `fh-js-sdk`

I would suggest fixing by setting `browserify-shim` as a `dependency` or `peerDependency` of `fh-js-sdk`. Thoughts?

**Comment by evanshortiss** [comment]
#211 opened as a suggested fix

**Comment by evanshortiss** [comment]
@wei-lee @wtrocki

**Comment by wtrocki** [comment]
@evanshortiss +1 on this.

 Do you think we need to also add something to README/Documentation?

**Comment by evanshortiss** [comment]
@wtrocki you mean I install them in my own project?

**Comment by wtrocki** [comment]
Yeah. That's ugly hack rather than real solution.

**Comment by evanshortiss** [comment]
Maybe I misunderstand, but I don't think my project should need to install the same `devDependencies` as `fh-js-sdk` - that's a much bigger hack. As a developer I think it's not unreasonable to expect to be able to `npm i fh-js-sdk` and have it work without needing to know extra steps. Do you think my suggestion of using `peerDependencies` is better?

Regardless, it seems like there's a weird bug with `browserify-shim` or `grunt-browserify`. If `browserify-shim` is not in `devDependencies` then running `npm run prepublish` fails 😱

**Comment by evanshortiss** [comment]
Ignore my previous comment. I missed the fact that we have `npm-shrinkwrap.json`. Fixing my PR!

**Comment by wtrocki** [comment]
Cool. I will verify if that's not breaking anything

**Comment by evanshortiss** [comment]
Thanks! Build should be good now 👍

**Comment by evanshortiss** [comment]
Closing due to fix being merged as #211
